pytify/cli.py

```diff
@@ -75,6 +77,8 @@ def interaction(self):
                 if search is not False:
                     self.menu(list=self.pytify.list())
 
+def autocomplete:
```

Comment:
Added parenthesis.

We need this function for the following reason:
- readline.parse_and_bind function expects and argument which is of the form <key>: <function>
-An alternative would to use tab: complete config which would use the rlcompleter default function and give a strange auto complete. Try it out.
-the auto complete function would be used later to implement the tab auto complete feature 

Great work on Pytify! :D